src/Tests/Tests.ScratchPad/Program.cs

```diff
@@ -1,11 +1,75 @@
-﻿using BenchmarkDotNet.Running;
-using Tests.ScratchPad.Runners.ApiCalls;
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Running;
+using Elasticsearch.Net;
+using Elasticsearch.Net.Diagnostics;
+using Nest;
 
 namespace Tests.ScratchPad
 {
 	public class Program
 	{
-		private static void Main(string[] args) => Bench<IndexDocumentRunner>();
+		private class ListenerObserver : IObserver<DiagnosticListener>
```

Comment:
I wonder if we could use this as an example for some docs?

Reply:
++ will amend with a `.doc.cs` test